esy.json

```diff
@@ -7,7 +7,7 @@
     "ocaml": " >= 4.2.0 < 4.9.0",
     "@opam/ocamlfind": "*",
     "@opam/menhir": " >= 20170418.0.0",
-    "@opam/utop": " >= 1.17.0 < 2.3.0",
+    "@opam/utop": " >= 1.17.0 < 2.5.0",
```

Comment:
I would even remove the upper bound constraint

Reply:
I think that upper bound was added because 2.3.1 was broken on Windows